mito-ai/src/Extensions/AiChat/ChatMessage/CodeBlock.tsx

```diff
@@ -65,12 +69,28 @@ const CodeBlock: React.FC<ICodeBlockProps> = ({
                     </div>
                     {isLastAiMessage && codeCellID !== undefined && (
                         <>
-                            <button onClick={() => { acceptAICode(codeCellID) }}>
-                                Apply {operatingSystem === 'mac' ? 'CMD+Y' : 'CTRL+Y'}
-                            </button>
-                            <button onClick={() => { rejectAICode(codeCellID) }}>
-                                Deny {operatingSystem === 'mac' ? 'CMD+D' : 'CTRL+D'}
-                            </button>
+                            {!isApplyingCode && !codeWasAccepted && (
+                                <button onClick={() => { setIsApplyingCode(true); applyAICode() }}>Apply</button>
+                            )}
+
+                            {isApplyingCode && !codeWasAccepted && (
+                                <>
+                                    <button
+                                        className='code-block-accept-button'
+                                        onClick={() => { acceptAICode(codeCellID); setIsApplyingCode(false); setCodeWasAccepted(true) }}
+                                        title={`Accept code (${operatingSystem === 'mac' ? 'CMD+Y' : 'CTRL+Y'})`}
+                                    >
+                                        Accept
+                                    </button>
+                                    <button
+                                        className='code-block-deny-button'
+                                        onClick={() => { rejectAICode(codeCellID); setIsApplyingCode(false) }}
+                                        title={`Deny code (${operatingSystem === 'mac' ? 'CMD+D' : 'CTRL+D'})`}
+                                    >
+                                        Deny
```

Comment:
Thoughts on adding the keyboard shortcuts back, but making them a bit less verbose by using the icons instead of CMD or CTRL. Like this? I think otherwise people might not discover it. 
![Screenshot 2024-12-04 at 3 10 38 PM](https://github.com/user-attachments/assets/f2ab6686-6d60-40f6-8d52-e814eff177ec)